package.json

```diff
@@ -15,7 +15,8 @@
     "packages/SwingSet",
     "packages/ERTP",
     "packages/cosmic-swingset",
-    "packages/agoric-cli"
+    "packages/agoric-cli",
+    "packages/zoe"
```

Comment:
I think this `"packages/zoe"` entry should go after its dependencies and before its dependents.  Not sure if it matters, though.